app/src/main/java/id/co/rolllpdf/presentation/dialog/LoadingDialog.kt

```diff
@@ -0,0 +1,71 @@
+package id.co.rolllpdf.presentation.dialog
+
+import android.animation.ObjectAnimator
+import android.content.Context
+import android.os.Handler
+import android.os.Looper.getMainLooper
+import android.view.View
+import android.view.animation.Interpolator
+import id.co.rolllpdf.core.Constant.FLOAT_ZERO
+import id.co.rolllpdf.core.Constant.ONE
+import id.co.rolllpdf.databinding.DialogLoadingBinding
+import id.co.rolllpdf.uikit.BaseDialog
+import kotlin.math.pow
+
+class LoadingDialog(context: Context, val title: String) : BaseDialog(context) {
+
+    private val binding by lazy {
+        DialogLoadingBinding.inflate(layoutInflater)
+    }
+
+    override fun setupLayout() {
+        val view = binding.root
+        setContentView(view)
+    }
+
+    override fun onCreateDialog() = with(binding) {
+        loadingTextviewTitle.text = title
+        loadingImageviewMascot.doBounceAnimation {
```

Comment:
duplicates with animation in splash screen. 
please create util class to handle this animation